### ISSUE TITLE
Error in graph feature creation

### ISSUE THREAD

**Issue by cbenitez81** [WangYueFt/dcp#9]
I check your code and I couldn't understand the graph feature with respect to dgcnn paper. It turns out that the original authors used to have _the bug_ in line 72 of your code 
`feature = torch.cat((feature, x), dim=3).permute(0, 3, 1, 2)` and it should be

`feature = torch.cat((feature - x, x), dim=3).permute(0, 3, 1, 2)`

Thanks for de code, I haven't tested, I'm still checking that it matches the theory

**Comment by WangYueFt** [comment]
> I check your code and I couldn't understand the graph feature with respect to dgcnn paper. It turns out that the original authors used to have _the bug_ in line 72 of your code
> `feature = torch.cat((feature, x), dim=3).permute(0, 3, 1, 2)` and it should be
> 
> `feature = torch.cat((feature - x, x), dim=3).permute(0, 3, 1, 2)`
> 
> Thanks for de code, I haven't tested, I'm still checking that it matches the theory

Thanks for pointing it out. Actually we tested both cases and didn't find critical differences.
